symphony/app/fbcnms-projects/magmalte/app/views/equipment/GatewayDetailConfig.js

```diff
@@ -285,108 +330,77 @@ function GatewayAggregation({gwInfo}: {gwInfo: lte_gateway}) {
-      readOnly: true,
-    },
-  };
+
+  const ran: KPIRows[] = [
+    [
+      {
+        category: 'PCI',
+        value: gwInfo.cellular.ran.pci,
+        statusCircle: false,
+      },
+      {
+        category: 'eNodeB Transmit',
+        value: gwInfo.cellular.ran.transmit_enabled ? 'Enabled' : 'Disabled',
+        statusCircle: false,
+      },
+    ],
+  ];
+
+  function ListItems(props) {
+    return (
+      <>
+        <ListItem>
+          <ListItemText primary={props.data} />
+        </ListItem>
+        <Divider />
+      </>
+    );
+  }
+
+  function ListNull() {
+    return (
+      <>
+        <ListItem>
+          <ListItemText primary="-" />
+        </ListItem>
+        <Divider />
+      </>
+    );
+  }
 
   return (
-    <List component={Paper}>
-      <ListItem>
-        <Grid container>
-          <Grid item xs={6}>
-            <TextField
-              fullWidth={true}
-              value={gwInfo.cellular.ran.pci}
-              label="PCI"
-              InputProps={{disableUnderline: true, readOnly: true}}
-            />
-          </Grid>
-          <Grid item xs={6}>
-            <TextField
-              fullWidth={true}
-              value={
-                gwInfo.cellular.ran.transmit_enabled ? 'Enabled' : 'Disabled'
-              }
-              label="eNodeB Transmit"
-              InputProps={{disableUnderline: true, readOnly: true}}
-            />
-          </Grid>
-        </Grid>
-      </ListItem>
+    <>
+      <KPIGrid data={ran} />
       <Divider />
-      <ListItem button onClick={() => setOpen(!open)}>
-        <ListItemText
-          primary={'Registered eNodeBs'}
-          secondary={gwInfo.connected_enodeb_serials?.length || 0}
-          {...typographyProps}
-        />
-        {open ? <ExpandLess /> : <ExpandMore />}
-      </ListItem>
-      <Collapse key="reservedIp" in={open} timeout="auto" unmountOnExit>
+      <List component={Paper} elevation={0} className={classes.list}>
+        <ListItem button onClick={() => setOpen(!open)}>
+          <CardHeader
+            title="Registered eNodeBs"
+            className={classes.kpiBox}
+            subheader={gwInfo.connected_enodeb_serials?.length || 0}
+            titleTypographyProps={{
+              variant: 'body3',
+              className: classes.kpiLabel,
+              title: 'Registered eNodeBs',
+            }}
+            subheaderTypographyProps={{
+              variant: 'body1',
+              className: classes.kpiValue,
+              title: gwInfo.connected_enodeb_serials?.length || 0,
+            }}
+          />
+          {open ? <ExpandLess /> : <ExpandMore />}
+        </ListItem>
         <Divider />
-        <Grid item xs={12}>
-          <EnodebsTable gwInfo={gwInfo} enbInfo={enbInfo} />
-        </Grid>
-      </Collapse>
-    </List>
+        <Collapse key="reservedIp" in={open} timeout="auto" unmountOnExit>
+          {gwInfo.connected_enodeb_serials?.map(data => (
```

Comment:
I made the list items non-interact-able (as in you can't click through to the listed eNodeB). If we want these listed eNodeB's to be something that users can interact with, we will most likely need some further work done or go with the Table approach, though the table will need some further work done to it from a styling perspective.

Anyway, I'd love to hear your thoughts on whether we need to do more than just list them out as I currently set it up to do.

Reply:
@HannaFar is this the connected eNodeB component ? if it is so then we would probably want to click through the eNodeB and land at the eNodeB page.

Reply:
It is. I'll revert this back to make use of that table and see whether or not I can fix the display of the table based on the design a bit better. It's work that needs to be done at some point anyway, so may as do it now.